wandb/sdk/artifacts/artifact.py

```diff
@@ -1783,6 +1806,7 @@ def _download(
         root: str,
         allow_missing_references: bool = False,
         skip_cache: Optional[bool] = None,
+        dir_path_prefix: Optional[StrPath] = None,
```

Comment:
Hm, I just realized - you'll have to implement the logic below in the sdk core too (nexus) - see `downloader.go`. I can help you trace the path.

Reply:
discussed on slack, this will be a separate PR